lib/core/card-view/components/card-view-selectitem/card-view-selectitem.component.ts

```diff
@@ -98,7 +98,7 @@ export class CardViewSelectItemComponent extends BaseCardView<CardViewSelectItem
     }
 
     showNoneOption() {
-        return this.displayNoneOption;
+        return this.displayNoneOption && this.property.key !== 'nodeType';
```

Comment:
this is a generic component we can not embed a business logic like node type here

Reply:
we need to pass displayNoneOption to false where we use it for the content type